File: ListaPOOExercicios/Atividade3/Lista1Atividade3-Java/src/ContaBancaria.java
public class ContaBancaria {
    private double saldo;
    private String titular;

    public ContaBancaria(double saldo, String titular){
        this.saldo = saldo;
        this.titular = titular;
    }

    public void depositar(double adepositar){
        this.saldo += adepositar;
    }
    public void sacar(double asacar){
        this.saldo -= asacar;
    }

    public void mostrarSaldo(){
        System.out.println("O seu Saldo é de "+saldo+"R$");
    }


}

File: ListaPOOExercicios/Atividade3/Python/Contabancaria.py
class Contabancaria:
    def __init__(self, saldo, titular):
        self.__saldo = saldo
        self.__titular = titular

    def sacar(self,dinheiroasacar):
        self.__saldo -= dinheiroasacar

    def depositar(self, dinheiroadepositar):
        self.__saldo += dinheiroadepositar

    def mostrarSaldo(self):
        print(f"Seu saldo é de: {self.__saldo} Reais")


cb = Contabancaria(100, "Eu")
print(cb._Contabancaria__saldo)     # Em python não há uma forma concreta de encapsulamento,
cb.mostrarSaldo()                   # as formas de proteger um atributo são por meio de convenções.
cb.sacar(100)
cb.mostrarSaldo()
cb.depositar(1000)
cb._Contabancaria__saldo += 50     # Forma errada de interagir com o saldo, segundo convenção
cb.mostrarSaldo()




